tests/dev/script-simulation-batch.R

```diff
@@ -0,0 +1,15 @@
+library(ospsuite)

```

Comment:
@PavelBal @StephanSchaller @Yuri05 
Here is a simple example of how to use the simulation batch